bundler/minify.js

```diff
@@ -1,69 +1,27 @@
 "use strict"
 
-var http = require("https")
-var querystring = require("querystring")
 var fs = require("fs")
+var UglifyES = require("uglify-es")
 
-module.exports = function(input, output, options, done) {
+module.exports = function(input, output, options) {
```

Comment:
`outputPath`/`inputPath` would be clearer here

Reply:
Agreed.